### PR TITLE
Add vanilla rspamd without redis

### LINKED ISSUE #71: Install rspamd
It can be done using in the redis role if we use a custom one (see #70 ).

### LINKED ISSUE #72: Integrate rspamd with postfix
It can be done using rspamd proxy worker and milters.
See https://rspamd.com/doc/integration.html

### PR DESCRIPTION
**Pull request by hugoShaka**
### Description
Install basic rspamd and integrate with postfix. No custom configuration yet.

Fixes #71, fixes #72

### Checklist
- [x] Molecule tests are passing
- [ ] Extended the README / documentation, if necessary
- [x] Test for the feature added

### CODE REVIEW

File: roles/ispmail-rspamd/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+- name: Add rspamd APT signing key
+  apt_key:
```
Comment (HadrienPatte) [inline roles/ispmail-rspamd/tasks/main.yml]: This won't pass ansible-lint 4.0.0 new rules, we should add something like:
```yaml
  register: apt_key_result
  retries: 3
  until: apt_key_result is succeeded
```

File: roles/ispmail-rspamd/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  loop:
+    - deb
+    - deb-src
+
+- name: Force trigger apt cache update if needed
+  meta: flush_handlers
+
+- name: Install rspamd packages
+  apt:
+    name:
```
Comment (HadrienPatte) [inline roles/ispmail-rspamd/tasks/main.yml]: As long as there is only one package to install, we can simplify this to:
```yaml
    name: rspamd
```

File: roles/ispmail-rspamd/tasks/main.yml (hunk 1 above)
Comment (hugoShaka) [reply roles/ispmail-rspamd/tasks/main.yml]: done

File: roles/ispmail-rspamd/tasks/main.yml (hunk 2 above)
Comment (hugoShaka) [reply roles/ispmail-rspamd/tasks/main.yml]: done

**Pull request merged**